hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncConfig.java

```diff
@@ -198,7 +198,7 @@ public TypedProperties toProps() {
       props.setPropertyIfNonNull(HIVE_SUPPORT_TIMESTAMP_TYPE.key(), supportTimestamp);
       props.setPropertyIfNonNull(HIVE_TABLE_PROPERTIES.key(), tableProperties);
       props.setPropertyIfNonNull(HIVE_TABLE_SERDE_PROPERTIES.key(), serdeProperties);
-      props.setPropertyIfNonNull(HIVE_SYNC_AS_DATA_SOURCE_TABLE.key(), syncAsSparkDataSourceTable);
+      props.setPropertyIfNonNull(HIVE_SYNC_AS_DATA_SOURCE_TABLE.key(), false);
```

Comment:
I don't remember why we explicitly set this to `false` but that's how it is in the `0.13.0` fork 🤷‍♂️